lib/Parser/Context/Default.pm

```diff
@@ -44,17 +44,18 @@ $operators = {
            class => 'Parser::BOP::add'},
 
    '-' => {precedence => 1, associativity => 'left', type => 'both', string => '-',
-           class => 'Parser::BOP::subtract', rightparens => 'same'},
+           class => 'Parser::BOP::subtract', rightparens => 'same', alternatives => ["\x{2212}"]},
 
    'U' => {precedence => 1.5, associativity => 'left', type => 'bin', isUnion => 1,
-           string => ' U ', TeX => '\cup ', class => 'Parser::BOP::union'},
+           string => ' U ', TeX => '\cup ', class => 'Parser::BOP::union',
+           alternatives => ["\x{222A}","\x{22C3}"]},
```

Comment:
Should `\x{22C3}` which is N-Ary Union be here? I'm not sure it is meant to be used as a binary operator (although if not, abuse is likely)
https://www.compart.com/en/unicode/U+22C3

Reply:
I suppose it depends on the philosophy behind allowing unicode characters to be used.  Does one want to map everything that people might use, or only the "correct" ones.  For example, do you map en-dash to minus (I know parsers that do), or just the dash and minus?  Do you map U+00B5 (micron sign) to U+03BC (Greek small letter mu)?

I'm not sure people looking for the union symbol will make the distinction between the correct one (U+222A) and the n-ary one (U+22C3), and certainly the original "U" (U+0055) is not correct.  

I'm OK with taking it out, if people think that is best, but was using it as an example of how to map multiple Unicode points as alternatives (there are other examples of that as well, so it is not critical).

Reply:
I also am not sure what is the right call here.
Abuse of the "wrong" one is likely and this is a "union" operator.